docs/terms/expectation_suite.md

```diff
@@ -3,12 +3,83 @@ id: expectation_suite
+Expectation Suites are *used* during the Validation of data.  In this step, you will need to provide one or more Expectation Suites to a Checkpoint.  This can either be done by configuring the Checkpoint to use a preset list of one or more Expectation Suites, or by configuring the Checkpoint to accept a list of one or more Expectation Suites at runtime.
+
+## Features
+
+### CRUD operations
+
+A Great Expectations Expectation Suite enables you to perform Create, Read, Update, and Delete (CRUD) operations on the Suite's Expectations without needing to re-run them. 
+
+### Reusability
+
+Expectation Suites are primarily used by Checkpoints, which can accept a list of one or more Expectation Suite.  Because they are stored independently of the Checkpoints that use them, the same Expectation Suite can be included in the list for multiple Checkpoints, provided the Expectation Suite contains a list of Expectations that describe the data that Checkpoint will Validate.  This allows you to, for example, create multiple Checkpoints with increasing vigor of in their validation, or create multiple Checkpoints with the same base set of Expectations and some additional Expectations tailored to specific data (such as one Checkpoint for validating raw data, and one with additional Expectation Suites for validating post-processing data).
+
+## API basics
+
+### CRUD operations
+
+Each of the Expectation Suite methods that support a Create, Read, Update, or Delete (CRUD) operation relies on two main parameters - `expectation_configuration` and `match_type`.
+
+- **expectation_configuration** - an `ExpectationConfiguration` object that is used to determine whether and where this Expectation already exists within the Suite. It can be a complete or a partial ExpectationConfiguration.
+- **match_type** - a string with the value of `domain`, `success`, or `runtime` which determines the criteria used for matching:
+    - `domain` checks whether two Expectation Configurations apply to the same data. It results in the loosest match, and can use the least complete ExpectationConfiguration object. For example, for a column map Expectation, `domain_kwargs` will include the Expectation_type, the column, and any row_conditions that may affect which rows are evaluated by the Expectation.
+    - `success` criteria are more exacting - in addition to the `domain` kwargs, these include those kwargs used when evaluating the success of an Expectation, like `mostly`, `max`, or `value_set`.
+    -`runtime` are the most specific - in addition to `domain_kwargs` and `success_kwargs`, these include kwargs used for runtime configuration. Currently, these include `result_format`, `include_config`, and `catch_exceptions`
+
+### How to access
+
+You will rarely need to directly access an Expectation Suite.  If you do need to edit one, the simplest way is through the CLI.  To do so, run the command:
+
+```markdown title="Terminal command"
+great_expectations suite edit NAME_OF_YOUR_SUITE_HERE
```

Comment:
I think we should mention the choice of whether to load data to edit the suite or not; I think there's a specific flag for that.

Reply:
I'll look into that.

Reply:
It looks like there is a `-ds, --datasource-name TEXT` flag.  is that the one you are thinking of?  Or the `-br, -batch-request TEXT` option?  (Perhaps both?)  Both are mutually exclusive and assume the use of the `--interactive` flag.

Reply:
I think the flag I was thinking of is the `--interactive` one...but I was thinking that was default. If that's not the cause (i.e. the default is to NOT apply interactive) then I think this is already correct as written.

Reply:
The default is interactive, but if you don't specify the CLI will prompt you to ask if you want to edit interactively or manually.